app/src/main/java/dmar/akan/akademiaandroida/features/characters/data/local/model/CharacterCached.kt

```diff
@@ -0,0 +1,49 @@
+package dmar.akan.akademiaandroida.features.characters.data.local.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import dmar.akan.akademiaandroida.features.characters.domain.model.Character
+
+@Entity
+data class CharacterCached(
+    @PrimaryKey
+    val id: Int,
+    val name: String,
+    val species: String,
+    val type: String,
+    val gender: String,
+    val origin: Character.CharacterOrigin,
+    val location: Character.CharacterLocation,
```

Comment:
Podobnie z kolejnym modelem, również używamy odpowiednika z warstwy danych.

```suggestion
    @Embeded("characterLocationCached")
    val location: CharacterLocationCached,
```